core/templates/pages/oppia_footer_directive.html

```diff
@@ -2,7 +2,7 @@
   <div class="oppia-footer-container">
     <div class="row row-cols-2">
       <div class="col">
-        <h4 translate="I18N_FOOTER_ABOUT_ALL_CAPS"></h4>
+        <div class="col-heading" translate="I18N_FOOTER_ABOUT_ALL_CAPS"></div>
```

Comment:
Changes in this file are for solving the failing lighthouse acccessibility test. As this part had some sequential tag problem.